src/config.rs

```diff
@@ -45,7 +57,7 @@ fn parse_cli_args() -> Result<CliArgs, getopts::Fail> {
     let args: Vec<String> = std::env::args().collect();
 
     let mut opts = getopts::Options::new();
-    opts.optopt("", "url", "URL to load on start", "URL");
+    opts.optopt("", "url", "URL to load on start", "docs.rs");
```

Comment:
I think it's better to keep URL as example instead.

Reply:
I wanted to change it to docs.rs because this show cases that you don't need a fully qualified URL for it to work